### PR TITLE
Rails backend

### PR DESCRIPTION
**Pull request by Tadessejemal1**
### Create a new Rails API app called 'hello-rails-back-end'.

- Initialize your project with Git.
- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route).
- Create a new React app called 'hello-react-front-end'
- Initialize your project with Git.

#

- Create the App component with react-router.
- Create the Greeting component that will display a greeting. Set it up as a route in your App component.
- Create a store, an action and a reducer that will connect to you API endpont to get the random greeting.
- Display the random greeting in your Greeting component.

#

- In the README file for the Rails API, add a link to your React app.
- When submitting for a review - use a link to the pull request in your Rails API.
- In the repo for the frontend part - include a PR with changes.
## unfortunately, I merged the PR with the dev branch if you want to see a pull request for the changes  please click the link here https://github.com/Tadessejemal1/Hello-react-frontend/pull/1

### CODE REVIEW

**Comment by mirouhml** [comment]
Hi @Tadessejemal1, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Please know that the frontend repository doesn't have an open Pull Request, you need to have two PRs one here and the other there.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Tadessejemal1** [comment]
## unfortunately, I merged the react app PR with the dev branch if you want to see a pull request for the changes. please click the link here https://github.com/Tadessejemal1/Hello-react-frontend/pull/1

File: db/seeds.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
+5.times do |i|
+    Greeting.create(
+      message: "Hello Tadesse #{i + 1}"
```
Comment (prolajumokeoni) [inline db/seeds.rb]: - [ ] Good job working on this project, Kindly create 5 random greeting messages that will be displayed, currently I see just 1-5 when I refresh 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Tadessejemal1 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used 
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly add the link of  the  frontend pr to the backend readme, so the next reviewer can easily navigate to the pr 👍🏿 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/seeds.rb (hunk 1 above)
Comment (Tadessejemal1) [reply db/seeds.rb]: Thank you for the feedback I fix it per your request.

**Review by julie-ify — APPROVED** [review]
Hi @Tadessejemal1,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**